tests/test_app/Plugin/Company/TestPluginThree/src/TestPluginThreePlugin.php

```diff
@@ -17,6 +17,6 @@
 
 use Cake\Core\BasePlugin;
 
-class Plugin extends BasePlugin
+class TestPluginThreePlugin extends BasePlugin
```

Comment:
oof, that name.

Reply:
You know what they say about naming things in programming :)